debian/control

```diff
@@ -232,6 +232,7 @@ Description: debugging symbols for ceph-mon
 Package: ceph-osd
 Architecture: linux-any
 Depends: ceph-base (= ${binary:Version}),
+         ceph-trace (= ${binary:Version}),
```

Comment:
if I add ceph-trace package in ceph-osd or other package's recommends fields, it also can be built into a ceph-tracexxx.deb package, but ceph-deploy can't install it automatically to /usr/lib.  if add it to depend field, ceph-deploy can install it automatically.